polyfill/index.d.ts

```diff
@@ -0,0 +1,385 @@
+    ): Temporal.Duration;
+    // Should Absolute and DateTime have different names for `inTimeZone`? See #574.
+    inTimeZone(temporalTimeZoneLike: TimeZoneLike, options?: ToAbsoluteOptions): Temporal.Absolute;
+    getDate(): Temporal.Date;
+    getYearMonth(): Temporal.YearMonth;
+    getMonthDay(): Temporal.MonthDay;
+    getTime(): Temporal.Time;
+    getFields(): Required<DateTimeLike>;
+    toLocaleString(locales?: string | string[], options?: Intl.DateTimeFormatOptions): string;
+    toJSON(): string;
+    toString(): string;
+  }
+
+  export type MonthDayLike = {
+    month?: number;
+    day?: number;
+  };
+
+  export class MonthDay extends Required<MonthDayLike> {
+    static from(item: Temporal.MonthDay | string | object, options?: AssignmentOptions): Temporal.MonthDay;
+    static compare(one: Temporal.MonthDay, two: Temporal.MonthDay): ComparisonResult;
+    constructor(month: number, day: number);
+    readonly month: number;
+    readonly day: number;
+    with(temporalMonthDayLike: MonthDayLike, options?: AssignmentOptions): Temporal.MonthDay;
+    withYear(year: number | { year: number }): Temporal.Date;
+    getFields(): Required<MonthDayLike>;
+    toLocaleString(locales?: string | string[], options?: Intl.DateTimeFormatOptions): string;
+    toJSON(): string;
+    toString(): string;
+  }
+
+  export type TimeLike = {
+    hour?: number;
+    minute?: number;
+    second?: number;
+    millisecond?: number;
+    microsecond?: number;
+    nanosecond?: number;
+  };
+
+  export class Time extends Required<TimeLike> {
+    static from(item: Temporal.Time | string | object, options?: AssignmentOptions): Temporal.Time;
+    static compare(one: Temporal.Time, two: Temporal.Time): ComparisonResult;
+    constructor(
+      hour?: number,
+      minute?: number,
+      second?: number,
+      millisecond?: number,
+      microsecond?: number,
+      nanosecond?: number
+    );
+    readonly hour: number;
+    readonly minute: number;
+    readonly second: number;
+    readonly millisecond: number;
+    readonly microsecond: number;
+    readonly nanosecond: number;
+    with(temporalTimeLike: TimeLike, options?: AssignmentOptions): Temporal.Time;
+    plus(temporalDurationLike: DurationLike, options?: ArithmeticOptions): Temporal.Time;
+    minus(temporalDurationLike: DurationLike, options?: ArithmeticOptions): Temporal.Time;
+    // TODO: should 'days' be included?  Or would that be false advertising? See #580.
+    difference(other: Temporal.Time, options?: DifferenceOptions<'hours' | 'minutes' | 'seconds'>): Temporal.Duration;
+    withDate(temporalDate: Temporal.Date): Temporal.DateTime;
+    getFields(): Required<TimeLike>;
+    toLocaleString(locales?: string | string[], options?: Intl.DateTimeFormatOptions): string;
+    toJSON(): string;
+    toString(): string;
+  }
+
+  export type TimeZoneLike = Temporal.TimeZone | string;
+
+  export class TimeZone {
```

Comment:
Heads up, I just merged `getPossibleAbsolutesFor(dateTime: Temporal.DateTime): Temporal.Absolute[];` into the main branch.

Reply:
Cool!  I'll update the types.

Reply:
Also, I noticed that `equals` was just committed. I'll add these methods into the TS types.